src/lolex-src.js

```diff
@@ -394,6 +398,12 @@ function hijackMethod(target, method, clock) {
     target[method].clock = clock;
 }
 
+exports.shouldAdvanceTime = false;
```

Comment:
I think a function with more than a couple of optional arguments is broken anyhow, especially when the optional ones can precede others like in `install`. I wonder if it would make sense to change the Lolex API for a version 2 now, before adding yet another config option to something like `lolex.install(config)`. That would scale with new options, such as this.